### PR TITLE
VideoPlayer: allow deinterlacing methods specific to renderer

### PR DESCRIPTION
**Pull request by FernetMenta**
allow deinterlacing methods specific to renderer

@afedchin this fix the issue you mentioned. Could you please update WinRenderer to add the deinterlacing methods?

### CODE REVIEW

**Comment by afedchin** [comment]
@FernetMenta I have PRed changes to your repo

**Comment by FernetMenta** [comment]
jenkins build this please

**Comment by FernetMenta** [comment]
@afedchin ok to merge?

**Comment by afedchin** [comment]
it's ok to go

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -8726,12 +8723,7 @@ msgstr ""
 
 #: xbmc/video/dialogs/GUIDialogVideoSettings.cpp
 msgctxt "#16320"
-msgid "DXVA - Bob"
-msgstr ""
-
-#: xbmc/video/dialogs/GUIDialogVideoSettings.cpp
-msgctxt "#16321"
-msgid "DXVA - Best"
+msgid "DXVA"
```
Comment (MartijnKaijser) [inline addons/resource.language.en_gb/resources/strings.po]: Line should be added saying string 16321 isn't used

Comment (FernetMenta) [inline addons/resource.language.en_gb/resources/strings.po]: updated

**Comment by FernetMenta** [comment]
jenkins build this please

**Pull request merged**